draft-ietf-quic-recovery.md

```diff
@@ -587,21 +562,18 @@ kTimeThreshold:
   considers a packet lost. Specified as an RTT multiplier. The RECOMMENDED
   value is 9/8.
 
-kMinTLPTimeout:
-: Minimum time in the future a tail loss probe timer may be set for.
-  The RECOMMENDED value is 10ms.
+kGranularity:
 
-kMinRTOTimeout:
-:  Minimum time in the future an RTO timer may be set for. The RECOMMENDED
-   value is 200ms.
+: Timer granularity. This is a system-dependent value.  However, implementations
+  SHOULD use a value no smaller than 1ms.
 
 kDelayedAckTimeout:
 : The length of the peer's delayed ack timer. The RECOMMENDED value is 25ms.
 
 kInitialRtt:
```

Comment:
Would you mind opening a separate issue for this? I'd argue for 100ms, but I don't want to do it in this PR. FWIW, I think the cost of an early timeout is not high, so having this be aggressive seems fine to me.

Reply:
Agreed, that's a separable issue, which was previously raised and then I believe closed.

Reply:
Ok I will open a separate issue. This will cause unnecessary timeouts for CI and 0-RTT data on many Internet paths. And pretty much always on satellite links. If we want to pick a magic number then let's pick 200 msec or 300 msec (the historical minrto values for Linux and Windows respectively).

Reply:
Here are 10 ping packets from Denmark to China

10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 211.218/350.768/593.271/149.383 ms

Reply:
And here from Denmark to New York, which is basically a direct sea cable link.

10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 103.982/104.505/105.343/0.475 ms

Reply:
And Denmark New Zealand

10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 324.042/454.764/672.469/122.684 ms